src/input/InputManager.cpp

```diff
@@ -40,13 +40,6 @@ CInputManager& CInputManager::Get(void)
   return _instance;
 }
 
-CInputManager::CInputManager()
-{
-  game_controller mouseControllerStruct = {};
-  mouseControllerStruct.controller_id = "game.controller.mouse";
-  m_mouseDevice = std::make_shared<CLibretroDevice>(&mouseControllerStruct);
-}
```

Comment:
It's not clear to me how this change is supposed to work. m_devices has no element for the mouse, because so far only OpenPort (via DeviceConnected) adds a new libretrodevice for it.